mpu6050_scripts/get_temperature.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#Init
+sudo i2cset -y 0 0x68 0x6B 0
+
+while true
+do
+
+#Read data
+UB=$(sudo i2cget -y 0 0x68 0x41)
+LB=$(sudo i2cget -y 0 0x68 0x42)
+
+#Upper Bit + Lower Bit
+TEMP=$(( (UB<<8)+LB-65535 ))
+
+# Calc C value
+TEMP_C=$(bc << EOF
```

Comment:
Is it possible to use bash operators using (( )) instead of calling bc.

Reply:
Like this: UB=$(sudo i2cget -y 0 0x68 0x41) && LB=$(sudo i2cget -y 0 0x68 0x42) && UBLB=$(sudo i2cget -y 0 0x68 0x41 w) && echo $UB $LB $UBLB